actioncable/lib/action_cable/server/base.rb

```diff
@@ -67,24 +67,14 @@ def worker_pool
         @worker_pool || @mutex.synchronize { @worker_pool ||= ActionCable::Server::Worker.new(max_size: config.worker_pool_size) }
       end
 
-      # Requires and returns a hash of all of the channel class constants, which are keyed by name.
-      def channel_classes
-        @channel_classes || @mutex.synchronize do
-          @channel_classes ||= begin
-            config.channel_paths.each { |channel_path| require channel_path }
-            config.channel_class_names.each_with_object({}) { |name, hash| hash[name] = name.constantize }
-          end
-        end
-      end
```

Comment:
I think it'll be in 5.0.0 (sorry @sgrif 😅 ) -- I'd claimed it was already working like this, and without it, all of ACa is clunky to use